### PR TITLE
documents to show pending on noi

### PR DESCRIPTION
**Pull request by urmikgov**

### CODE REVIEW

File: alcs-frontend/src/app/features/notice-of-intent/documents/documents.component.scss [hunk 1]
```diff
@@ -17,3 +17,20 @@
...
+table {
+  position: relative; 
+
+  th mat-header-cell {
+    position: relative; 
+  }
+
+  .subheading {
+    font-size: 11px;
+    line-height: 16px;
```
Comment (trslater) [inline alcs-frontend/src/app/features/notice-of-intent/documents/documents.component.scss:30]: Should these be using the `rem` function?

Comment (urmikgov) [reply alcs-frontend/src/app/features/notice-of-intent/documents/documents.component.scss:30]: its kept common from alcs-frontend/src/app/features/application/decision/decision-v2/decision-documents/decision-documents.component.scss

Comment (dhaselhan) [reply alcs-frontend/src/app/features/notice-of-intent/documents/documents.component.scss:30]: We use rem in the portal but px in ALCS. ALCS is not responsive whereas the Portal is.

**Review by dhaselhan — APPROVED** [review]
LGTM 👍

**Pull request merged**